src/main/resources/migrations/00029_alter_table_gocardless_events.sql

```diff
@@ -0,0 +1,13 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:alter_table-gocardless-events-internal-id
+ALTER TABLE gocardless_events ADD COLUMN internal_event_id BIGINT;
```

Comment:
why not call it `event_id`?

Reply:
because there is already an event_id which is currently used as a gocardless_event_id :(